backend/store/error_groups.go

```diff
@@ -43,6 +43,11 @@ func (store *Store) ListErrorObjects(errorGroup model.ErrorGroup, params ListErr
 					Where("sessions.email ILIKE ?", "%"+val[0]+"%")
 			}
 		}
+		if val, ok := filters.Attributes["has_session"]; ok {
+			if len(val) > 0 && val[0] == "true" {
+				query.Where("COALESCE(error_objects.session_id) > 0")
```

Comment:
do you want `COALESCE(error_objects.session_id, 0)` here? I think coalesce will still return null if its last argument is null

Reply:
thanks! forgot about that